SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,7 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||bcvideod.com/view|
```

Comment:
I can't find this request too

Reply:
@Sergey-Lyapin You're right, since both are loaded from `http://peretrah.com/jscba3/7499af195420de.js` blocked by Russian Filter I should have not seen them, I guess I first tested without Russian Filter and forgot to clear cache. I'll remove them soon.